foundry.toml

```diff
@@ -1,7 +1,7 @@
 [profile.default]
-src = 'src'
-test = 'tests'
-script = 'scripts'
+src = 'src/contracts'
```

Comment:
just curious, why does it matter?

Reply:
per foundry docs, it references the path to contract sources. Not sure if a wrong config can bring any issue